### PR TITLE
[SYCLomatic][NFC]Add 'verify the installation' in readme.md

### PR DESCRIPTION
**Pull request by wenhuiNi**
Signed-off-by: Ni, Wenhui <wenhui.ni@intel.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,23 @@ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%PATH_TO_C2S_INSTALL_FOLDER%  -DCMAKE_BUIL
 ninja install-c2s
 ```
 
-### Deployment
+Note:For computers with 16G memory, we recommend using 4-6 threads for compilation. Using too many threads may cause out of memory.
```
Comment (zhiweij1) [inline README.md]:
```suggestion
Note: For computers with memory less than 16GB, we recommend using 4-6 threads for compilation. Using too many threads may cause out of memory.
```

File: README.md [hunk 2]
```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
 SET CPATH=%PATH_TO_C2S_INSTALL_FOLDER%\include;%CPATH%
 ```
 
-### Test SYCLomatic
-#### Run in-tree LIT tests
+### Verify the Installation
+Before run SYCLomatic， it is important to verify that dpct/c2s(SYCLomatic binary) can be find in terminal.
```
Comment (zhiweij1) [inline README.md]:
```suggestion
Before running SYCLomatic, it is important to verify that dpct/c2s(SYCLomatic binary) can be found in terminal.
```

File: README.md [hunk 3]
```diff
@@ -102,7 +102,23 @@ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%PATH_TO_C2S_INSTALL_FOLDER%  -DCMAKE_BUIL
...
-### Deployment
+Note:For computers with 16G memory, we recommend using 4-6 threads for compilation. Using too many threads may cause out of memory.
+```
+ninja -j4 install-c2s
+```
+
+
+**build success message**:
+
+After the compilation is successful, the following information will be output in terminal.
```
Comment (zhiweij1) [inline README.md]:
```suggestion
After the compilation is successful, the following information will be outputted in terminal.
```

File: README.md [hunk 4]
```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
 SET CPATH=%PATH_TO_C2S_INSTALL_FOLDER%\include;%CPATH%
 ```
 
-### Test SYCLomatic
-#### Run in-tree LIT tests
+### Verify the Installation
+Before run SYCLomatic， it is important to verify that dpct/c2s(SYCLomatic binary) can be find in terminal.
+
+To do this, you need run the following command and get correctly output.
```
Comment (zhiweij1) [inline README.md]:
```suggestion
To do this, you need run the following command and get correct output.
```

File: README.md [hunk 5]
```diff
@@ -102,7 +102,23 @@ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%PATH_TO_C2S_INSTALL_FOLDER%  -DCMAKE_BUIL
 ninja install-c2s
 ```
 
-### Deployment
+Note: For computers with 16G and below memory, we recommend using 4-6 threads for compilation. Using too many threads may cause out of memory.
```
Comment (AndyCHHuang) [inline README.md]: For computers with 16G or less memory, we recommend using 4-6 threads for compilation. Compiling SYCLomatic with too many threads may cause out of memory.

File: README.md [hunk 6]
```diff
@@ -102,7 +102,23 @@ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%PATH_TO_C2S_INSTALL_FOLDER%  -DCMAKE_BUIL
...
+```
+ninja -j4 install-c2s
+```
+
+
+**build success message**:
+
+After the compilation is successful, the following information will be outputted in terminal.
```
Comment (AndyCHHuang) [inline README.md]: After a successful build, you should be able to see following message in the terminal output.

File: README.md [hunk 7]
```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
...
+To do this, you need run the following command and get correct output.
+```
+dpct --version
+```
+Note: Ensure the PATH and CPATH are set correctly.
+
+## Run SYCLomatic
+### Run c2s command
+Get c2s help information by running "c2s --help".
+dpct is an alias command for c2s.
```
Comment (AndyCHHuang) [inline README.md:148]: c2s is an alias command for dpct.

File: README.md [hunk 8]
```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
...
+### Verify the Installation
+Before running SYCLomatic, it is important to verify that dpct/c2s(SYCLomatic binary) can be found in terminal.
+
+To do this, you need run the following command and get correct output.
+```
+dpct --version
+```
+Note: Ensure the PATH and CPATH are set correctly.
+
+## Run SYCLomatic
```
Comment (AndyCHHuang) [inline README.md:145]: ## Run SYCLomatic
### Run dpct command
The help information of dpct is available by running
```
dpct --help
```

File: README.md [hunk 9]
```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
 SET CPATH=%PATH_TO_C2S_INSTALL_FOLDER%\include;%CPATH%
 ```
 
-### Test SYCLomatic
-#### Run in-tree LIT tests
+### Verify the Installation
+Before running SYCLomatic, it is important to verify that dpct/c2s(SYCLomatic binary) can be found in terminal.
+
+To do this, you need run the following command and get correct output.
```
Comment (AndyCHHuang) [inline README.md]: Before running SYCLomatic, it is important to verify that dpct/c2s(SYCLomatic binary) is installed correctly and PATH is set properly.
To do this, try running the following command and checking if dpct is available after the installation.

File: README.md [hunk 10]
```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
...
+```
+dpct --version
+```
+Note: Ensure the PATH and CPATH are set correctly.
```
Comment (AndyCHHuang) [inline README.md]: If dpct cannot be found, please check the setup of PATH.

File: README.md [hunk 11]
```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
...
+
+## Run SYCLomatic
+### Run c2s command
+Get c2s help information by running "c2s --help".
+dpct is an alias command for c2s.
+
+
+## Test SYCLomatic (Recommended to Contributors)
```
Comment (AndyCHHuang) [inline README.md]: ## Test SYCLomatic

File: README.md [hunk 12]
```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
...
+### Run in-tree LIT tests
 
 Note: Certain CUDA header files may need to be accessible to the tool.
 After build the SYCLomatic, you can run the list test by: 
 
 ninja check-clang-c2s
 
 
-#### Run E2E test suite
+### Run E2E test suite
```
Comment (AndyCHHuang) [inline README.md]: ### Run E2E test suite (Recommended to Contributors)

File: README.md [hunk 13]
```diff
@@ -102,7 +102,23 @@ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%PATH_TO_C2S_INSTALL_FOLDER%  -DCMAKE_BUIL
 ninja install-c2s
 ```
 
-### Deployment
+Note: For computers with 16G or less memory, we recommend using 4-6 threads for compilation. Compiling SYCLomatic with too many threads may cause out of memory.
```
Comment (zhimingwang36) [inline README.md]:
```suggestion
Note: For build system with 16G or less memory, recommend to use less than 5 threads for compilation,  you may meet out of memory issue when more than 4 threads used for compilation.
```

File: README.md [hunk 14]
```diff
@@ -102,7 +102,23 @@ cmake -G Ninja -DCMAKE_INSTALL_PREFIX=%PATH_TO_C2S_INSTALL_FOLDER%  -DCMAKE_BUIL
...
+ninja -j4 install-c2s
+```
+
+
+**build success message**:
+
+After a successful build, you should be able to see following message in the terminal output.
+```
+-- Install configuration: "Release"
+-- Installing: $PATH_TO_C2S_INSTALL_FOLDER/bin/dpct
```
Comment (zhimingwang36) [inline README.md]:
```suggestion
-- Installing: ... bin/dpct
```

File: README.md [hunk 15]
```diff
@@ -117,26 +133,35 @@ SET INCLUDE=%PATH_TO_C2S_INSTALL_FOLDER%\include;%INCLUDE%
...
+### Run in-tree LIT tests
 
 Note: Certain CUDA header files may need to be accessible to the tool.
 After build the SYCLomatic, you can run the list test by: 
 
 ninja check-clang-c2s
 
 
-#### Run E2E test suite
+### Run E2E test suite (Recommended to Contributors)
```
Comment (zhimingwang36) [inline README.md]:
```suggestion
### Run E2E test suite (Recommend for contributors)
```

**Review by zhiweij1 — APPROVED** [review]

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**